### ISSUE TITLE
Android support

### ISSUE THREAD

**Issue by jforaker** [alawong/react-native-braintree#1]
Plans?

**Comment by alanhhwong** [comment]
@jforaker currently trying to focus on getting adequate features in for iOS first, and then move on to Android

**Comment by SeraphimSerapis** [comment]
Hey @jforaker I'll happily take care of this one. Thanks for the suggestion!

**Comment by praisegeek** [comment]
+1 for Android support

**Comment by sulliwane** [comment]
Any chance to merge with this: https://github.com/surialabs/react-native-braintree-android for android support.
:+1:

**Comment by rstrobl** [comment]
@alawong hi, greetings from KL :) good to know that others in our area also doing react native.
wanna have a chat about how to get this merged?

**Comment by alanhhwong** [comment]
@rstrobl absolutely.  do you want to shoot me an email and we can schedule something?  alan.wong@braintreepayments.com

**Comment by ecjs** [comment]
Are there any plans to get these integrated sometime soon?

**Comment by rstrobl** [comment]
hey guys... sorry, was very busy the last 2 months. i will address some issues that were reported on the android site the next days and get back to you @alawong 

for now, you can still use both gems and use conditionals to check for the OS. it's just a few LOC. i know it'd be nicer to have this merged though. will keep you posted.

**Comment by kraffslol** [comment]
Hello everyone! Since no progress has been made in almost 4 months in regards of merging, I've decided to merge the modules myself. If any future reader is seeing this you can try the cross-platform module [here](https://github.com/kraffslol/react-native-braintree-xplat).

**Comment by rstrobl** [comment]
great, thanks @kraffslol ... do you guys think you can find a way on having this run under the `react-native-braintree` package name, e.g. sending a PR to this repository or handing over the npm repository to krafflol for further maintenance? cc @alawong

**Comment by kraffslol** [comment]
Sadly I couldn't make a PR because I couldn't get the current setup to work with rnpm, and I don't know enough about Cocoapods to sort that out. I think it's a good idea to keep all the 3 packages separate so we don't risk breaking the current installations of the modules. Maybe point towards to my package as an alternative if one would want cross-platform support?
